experimental/packages/opentelemetry-sdk-metrics-base/test/aggregator/LastValue.test.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HrTime } from '@opentelemetry/api';
+import { AggregationTemporality } from '@opentelemetry/api-metrics-wip';
+import * as assert from 'assert';
+import { LastValueAccumulation, LastValueAggregator } from '../../src/aggregator';
+import { MetricData, PointDataType } from '../../src/export/MetricData';
+import { commonValues, defaultInstrumentationLibrary, defaultInstrumentDescriptor, defaultResource } from '../util';
+
+describe('LastValueAggregator', () => {
+  describe('createAccumulation', () => {
+    it('no exceptions on createAccumulation', () => {
+      const aggregator = new LastValueAggregator();
+      const accumulation = aggregator.createAccumulation();
+      assert(accumulation instanceof LastValueAccumulation);
+    });
+  });
+
+  describe('merge', () => {
+    it('no exceptions', () => {
+      const aggregator = new LastValueAggregator();
+      const prev = aggregator.createAccumulation();
+      prev.record(2);
+
+      const delta = aggregator.createAccumulation();
+      delta.record(3);
+
+      assert.deepStrictEqual(aggregator.merge(prev, delta), delta);
+    });
+  });
+
+  describe('diff', () => {
+    it('no exceptions', () => {
+      const aggregator = new LastValueAggregator();
+      const prev = aggregator.createAccumulation();
+      prev.record(2);
+
+      const curr = aggregator.createAccumulation();
+      curr.record(3);
+
+      assert.deepStrictEqual(aggregator.diff(prev, curr), curr);
+    });
+  });
+
+  describe('toMetricData', () => {
+    it('transform with AggregationTemporality.DELTA', () => {
+      const aggregator = new LastValueAggregator();
+
+      const accumulation = aggregator.createAccumulation();
+      accumulation.record(1);
+      accumulation.record(2);
+      accumulation.record(1);
```

Comment:
Maybe we should use 1,2, 4 in order to show that it is actually the last value thats exported (in this case it could also be the first value, and there is no way to tell them apart).